.github/workflows/misc-sync-templates.yml

```diff
@@ -83,6 +83,12 @@ jobs:
           homepage = "https://paritytech.github.io/polkadot-sdk/"
 
           [workspace]
+          EOF
```

Comment:
@EgorPopelyaev is this workflow called in the release process, or should we do it manually? 

The person who made them has left, so a bit worried that it will go stale.

Reply:
It must be called manually: https://github.com/paritytech/polkadot-sdk/blob/master/.github/workflows/misc-sync-templates.yml#L18

Reply:
Created this https://github.com/paritytech/polkadot-sdk/issues/6329.